src/main/java/lotto/exception/BuyLottoMoneyFormatException.java

```diff
@@ -0,0 +1,9 @@
+package lotto.exception;
+
+public class BuyLottoMoneyFormatException extends IllegalArgumentException {
+    private static final String ERROR_MESSAGE = "[ERROR] 1,000원 단위의 금액을 입력해야합니다.";
```

Comment:
`IllegalArgumentException`을 상속한 각각의 커스텀 예외들은 모두 메시지로 "[ERROR]"를 포함하고 있는데요.
이 경우 해당 메시지의 세부 사항이 변경되었을 때 일일이 수정을 해 주어야 하는 단점이 있을 것 같아요.
해당 값을 별도의 상수 클래스로 분리해 사용하면 좋을 것 같습니다.

```suggestion
    private static final String ERROR_MESSAGE = LottoConst.ERROR_PREFIX + "1,000원 단위의 금액을 입력해야합니다.";
```